services/web/server/src/simcore_service_webserver/projects/_controller/groups_rest.py

```diff
@@ -1,6 +1,11 @@
 import logging
 
 from aiohttp import web
+from models_library.api_schemas_webserver.groups import (
+    ProjectShare,
+    ProjectShareAccepted,
+)
```

Comment:
this shows that the location of this model is maybe not as it should be.

Reply:
refactored.